service-config/src/main/java/com/palantir/config/service/ServiceDiscoveryConfiguration.java

```diff
@@ -136,6 +136,12 @@
         return getServiceConfig(serviceName).readTimeout();
     }
 
+    public final boolean isServiceEnabled(String serviceName) {
```

Comment:
See the other reply :)


> On Apr 29, 2016, at 12:51 AM, Robert Fink <notifications@github.com> wrote:
> 
> In service-config/src/main/java/com/palantir/config/service/ServiceDiscoveryConfiguration.java:
> 
> > @@ -136,6 +136,12 @@
> >          return getServiceConfig(serviceName).readTimeout();
> >      }
> >  
> > +    public final boolean isServiceEnabled(String serviceName) {
> how about: hasService()?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>